addons/account/models/account_account.py

```diff
@@ -747,7 +747,9 @@ def _unlink_except_account_set_on_customer(self):
         if partner_prop_acc:
             account_name = partner_prop_acc.get_by_record().display_name
             raise UserError(
-                _('You cannot remove/deactivate the account %s which is set on a customer or vendor.', account_name)
+                _("You can't delete the account %s, as it is used on a contact.\n\n"
+                    "Imagine that a customer’s receivables evaporate; your CFO might not like it :)"
```

Comment:
It's unfortunate that I was not aware of this PR, otherwise I would have added to this review before it was r+ed. 
I have to agree completely with what Tiffany is saying here. 

The sense of the word "evaporate" in this context does not translate well. I think Tiffany's suggestion is much better, "vanishing" would also work. 

Also, Tiffany is right about the semicolons. They're generally regarded as a more "technical" piece of punctuation, and are thus more formal in nature.

Reply:
This part is basically all you need: _("You can't delete the account %s since it is used by a contact.\n\n"  Not all companies have CFO. And the result of blocking delete accounts are much more serious than the mood of the CFO.